src/app.js

```diff
@@ -25,22 +27,25 @@ if(getEnv() === 'development') {
       chunkModules: false,
       modules: false
     }
-  }));
-  app.use(require('webpack-hot-middleware')(compiler));
+  }))
+  app.use(require('webpack-hot-middleware')(compiler))
 }
 
-app.use(logger('dev'));
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(cookieParser());
+app.use(logger('dev'))
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(cookieParser())
 app.use(express.static(path.join(__dirname, 'public')))
 auth(app)
 
 app.use('/api/task', task)
+app.use('/noob', noob)
```

Comment:
also can you make the routes plural? so `admin` becomes `admins`, `noob` becomes `noobs` etc